### PR TITLE
Updating job openings

### PR DESCRIPTION
**Pull request by burtonjc**
1. Include the opening for a devops engineer
1. incorporate feedback on how we describe what the positions are

Let's do the same thing, I'll wait for everyone to approve before I merge.

### CODE REVIEW

File: CAREERS.md [hunk 1]
```diff
@@ -49,16 +54,46 @@ We use our own product to manage our work. Anyone in the world can see what we
...
+- Managing production and development infrastructure using Docker, Terraform, Kubernetes, GKE, HAProxy, and NodeJS
+- Owning monitoring and instrumentation of each of the Waffle services
+- Building chatops to support your work and the rest of the team
+- Managing MongoDB hosted by Compose.com
+- Maintaining and advancing CI/CD via CircleCI
+- Working with the rest of the team to determine and execute on future operational and infrustructural changes
+
+What you'll bring:
+
+We are looking for passionate, self-directed, people who care about the customers their software is serving. For this possition we are seeking an individual with solid experience and any of the following will give you a good head start:
```
Comment (marydavis) [inline CAREERS.md]: 'position'

File: CAREERS.md [hunk 2]
```diff
@@ -49,16 +54,46 @@ We use our own product to manage our work. Anyone in the world can see what we
...
+- Solid expertise in networking, both in the cloud an in on-premesis networks
```
Comment (marydavis) [inline CAREERS.md]: "in the cloud and in on-premises'

File: CAREERS.md [hunk 3]
```diff
@@ -49,16 +54,46 @@ We use our own product to manage our work. Anyone in the world can see what we
...
-- are experienced with MongoDB.
-- have experience designing, monitoring, and scaling web services.
-- enjoy working with others to solve problems. At Waffle we often pair on work and are always collaborating for the best solution.
-- enjoy understanding how customers use the code you write. At Waffle we are all about solving the right problem for the right customer, and it is everyone's job to put the customer first.
-- have contributed to open source. We are widely adopted by the open source community and we love candidates who give back!
+We're hiring for a **senior devops engineer** and two **full stack senior software engineers** to join our team in Boulder, CO or Raleigh, NC!
+
+#### Senior DevOps Engineer
+What you'll be doing:
+- Working closesly with the whole Waffle team to drive the Waffle mission forward and make our customers most successful
```
Comment (AnaMSteele) [inline CAREERS.md]: typo change "closesly" to "closely"

File: CAREERS.md [hunk 4]
```diff
@@ -49,16 +54,46 @@ We use our own product to manage our work. Anyone in the world can see what we
...
+- Supporting and advancing [Waffle Takeout](https://takeout.waffle.io), the on premises installation of Waffle
+- Managing production and development infrastructure using Docker, Terraform, Kubernetes, GKE, HAProxy, and NodeJS
+- Owning monitoring and instrumentation of each of the Waffle services
+- Building chatops to support your work and the rest of the team
+- Managing MongoDB hosted by Compose.com
+- Maintaining and advancing CI/CD via CircleCI
+- Working with the rest of the team to determine and execute on future operational and infrustructural changes
```
Comment (AnaMSteele) [inline CAREERS.md]: I believe "infrastructure" should be used here

File: CAREERS.md [hunk 5]
```diff
@@ -49,16 +54,46 @@ We use our own product to manage our work. Anyone in the world can see what we
...
-- are an expert with NodeJS and JavaScript.
-- are experienced with Angular and AngularJS.
-- are experienced with MongoDB.
-- have experience designing, monitoring, and scaling web services.
-- enjoy working with others to solve problems. At Waffle we often pair on work and are always collaborating for the best solution.
-- enjoy understanding how customers use the code you write. At Waffle we are all about solving the right problem for the right customer, and it is everyone's job to put the customer first.
-- have contributed to open source. We are widely adopted by the open source community and we love candidates who give back!
+We're hiring for a **senior devops engineer** and two **full stack senior software engineers** to join our team in Boulder, CO or Raleigh, NC!
+
+#### Senior DevOps Engineer
```
Comment (GreenEyez86) [inline CAREERS.md]: Correct typo (DevOps to Develop)

**Pull request merged**